src/_components/shared/navbar.erb

```diff
@@ -0,0 +1,15 @@
+<header>
+  <a href="/">
+    <img src="<%= relative_url '/images/bump-logo.svg' %>" alt="Bump.sh" />
+  </a>
+
+  <nav data-controller="toggle">
+    <%= liquid_render "bridgetown_quick_search/search", placeholder: "Search…" %>
```

Comment:
Is it safe to mix liquid and rails rendering ?

Reply:
The `liquid_render` is a helper provided by Bridgetown so I assume this is safe, yes.
Also I don't have another way to render the search component as the plugin only exists in the liquid version.